HLTrigger/JSONMonitoring/plugins/TriggerJSONMonitoring.cc

```diff
@@ -573,6 +579,14 @@ TriggerJSONMonitoring::globalEndLuminosityBlockSummary(const edm::LuminosityBloc
     outHltJsnData<<result;
     outHltJsnData.close();
 
+    //HLT jsn entries
+    StringJ hltJsnFilelist_;
```

Comment:
hi Srecko,
please do not use names ending with an underscore for non-member variables.

.A